### PR TITLE
EPIC ess 1240

### PR DESCRIPTION
**Pull request by leonardotemasys**

### CODE REVIEW

File: demo/assets/css/calls.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.peers-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding: 20px 20px 20px 20px;
```
Comment (ishannaithani) [inline demo/assets/css/calls.css]: just padding: 20px should do?

File: demo/assets/css/calls.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    width: 200px;
+    position: relative;
+    left: 0%;
+    top: 7%;
+}
+
+.peer-name {
+    font-size: 0.8rem;
+    font-weight: bold;
+    padding: 10px 10px 10px 10px;
```
Comment (ishannaithani) [inline demo/assets/css/calls.css]: padding: 10px

File: demo/assets/css/base.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    font-size: 1rem;
+}
+
+/* TABLES */
+.table {
+    border: 1px solid;
+    width: 100%;
+}
+
+th {
```
Comment (ishannaithani) [inline demo/assets/css/base.css]: th, tr, td could be scoped inside the .table? .table th {}  & .table tr {} & .table tr td {}

File: demo/chat-room/style.css [hunk 4]
```diff
@@ -1,151 +1,111 @@
...
+    resize: none;
+}
+
+.highlight {
+    background-color: #f7f7f9;
+    border: 1px solid #e1e1e8;
+}
+
+.send-message-btn {
+    border-top-left-radius: 0;
```
Comment (ishannaithani) [inline demo/chat-room/style.css]: border-radius: 0 could work here

**Comment by ishannaithani** [comment]
Looks good

File: demo/chat-room/style.css (hunk 4 above)
Comment (leonardotemasys) [reply demo/chat-room/style.css]: It does not because we only need the upper part

File: demo/assets/css/base.css (hunk 3 above)
Comment (leonardotemasys) [reply demo/assets/css/base.css]: Can

File: demo/assets/css/calls.css (hunk 2 above)
Comment (leonardotemasys) [reply demo/assets/css/calls.css]: Ok, Can

File: demo/assets/css/calls.css (hunk 1 above)
Comment (leonardotemasys) [reply demo/assets/css/calls.css]: I'll try

File: demo/assets/css/calls.css (hunk 2 above)
Comment (leonardotemasys) [reply demo/assets/css/calls.css]: Implemented

**Pull request merged**